src/Common/src/Interop/Unix/libssl/SecuritySafeHandles.cs

```diff
@@ -56,12 +56,19 @@ protected override bool ReleaseHandle()
     // Implementation of handles dependable on FreeCredentialsHandle
     //
 #if DEBUG
-    internal sealed class SafeFreeCredentials : DebugSafeHandle
+    internal abstract class SafeFreeCredentials : DebugSafeHandle
```

Comment:
As post-merge cleanup this file should probably be split up (now that this base class applies to two different libraries' interop).  And since `libssl` is the last `lib*` directory left in Interop, that could be cleaned up at the same time.